openedx_webhooks/github/dispatcher/rules/github_activity.py

```diff
@@ -21,6 +19,8 @@ def _find_issues_for_pull_request(pull_request_url, jira=jira):
     Returns:
         jira.client.ResultList[jira.Issue]
     """
+    if jira is None:
+        from ....lib.jira.client import jira
```

Comment:
I guess I would deal with that problem closer to where the environment variables are being read, rather than expecting the users of the code to remember that they need to make local imports.